bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -36,72 +36,40 @@
     """
     )
 
-try:
-    import tensorflow_hub as hub  # type: ignore
-    from tensorflow_hub import resolve  # type: ignore
-    from tensorflow_hub import native_module  # type: ignore
-except ImportError:  # pragma: no cover
-    logger.warning(
-        """\
-    If you want to use `bentoml.tensorflow.import_from_tfhub(),
-     make sure to `pip install --upgrade tensorflow_hub` before using.
-     """
-    )
-    hub = None
-
-
-try:
-    import importlib.metadata as importlib_metadata
-except ImportError:
-    import importlib_metadata
-
 
 if TYPE_CHECKING:
-    from tensorflow_hub import Module as HubModule  # type: ignore
-    from tensorflow_hub import KerasLayer  # type: ignore
-
     from .. import external_typing as ext
-    from ..types import PathType
-    from ..models import ModelStore
+    from ..models.model import ModelSignatureDict
     from ..external_typing import tensorflow as tf_ext
 
     TFArgType = t.Union[t.List[t.Union[int, float]], ext.NpNDArray, tf_ext.Tensor]
 
-MODULE_NAME = "bentoml.tensorflow_v2"
+MODULE_NAME = "bentoml.tensorflow"
 
 
-def _clean_name(name: str) -> str:  # pragma: no cover
-    if name.startswith(("http://", "https://")):
-        name = name.split("/", maxsplit=3)[-1]
-    else:
-        name = name.split("/")[-1]
-    return re.sub(r"\W|^(?=\d)-", "_", name)
+def get(tag_like: str | Tag) -> bentoml.Model:
+    model = bentoml.models.get(tag_like)
```

Comment:
```suggestion
    model = get(tag_like)
```
```suggestion
    model = bentoml.models.get(tag_like)
```

Reply:
this line cannot use `get` because `get` is not defined yet